### ISSUE TITLE
Validation Error Jittering

### ISSUE THREAD

**Issue by lgyhero** [vlfeat/matconvnet#469]
![image](https://cloud.githubusercontent.com/assets/3654356/13898660/677b32c0-ee13-11e5-8368-3ba1152fef4c.png)
I am using Matconvnet to do a binary-classfication job, the train error is decreasing smoothly, but the validation error is jittering....Can anyone explain this? Thanks a lot.

**Comment by vedaldi** [comment]
Seems perfectly normal. Probably your validation set is much smaller than the training set, so there is a bit of statistical fluctuation.

A

**Comment by lgyhero** [comment]
Got it. I'v divided the dataset(about 200,000 images) into 80% training set and 20% validation set. Maybe I can offer more images for validation. Thanks very much !@vedaldi 
I'll close the issue.
